client/my-sites/plans-v2/more-info-box/index.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Button } from '@automattic/components';
+
+/**
+ * Type dependencies
+ */
+import type { MoreInfoProps } from 'calypso/my-sites/plans-v2/types.ts';
```

Comment:
We could probably define types in this file, for better readability, since they're not imported anywhere else.

Reply:
Ahh yes, ok got it.  Thank you! All this TypeScript is pretty new to me. 😉  👍